plugin/build.gradle

```diff
@@ -123,11 +122,21 @@ dependencies {
     api "com.fasterxml.jackson.core:jackson-core:${versions.jackson}"
     api "com.fasterxml.jackson.core:jackson-databind:${versions.jackson_databind}"
     api "com.fasterxml.jackson.core:jackson-annotations:${versions.jackson}"
+    api group: 'commons-io', name: 'commons-io', version: '2.8.0'
+    implementation group: 'org.opensearch', name: 'opensearch-x-content', version: "${opensearch_version}"
+    implementation group: 'org.opensearch', name: 'opensearch-ml-client', version: "${opensearch_build}"
+    implementation group: 'org.opensearch', name: 'common-utils', version: "${opensearch_build}"
```

Comment:
why depende on ml-client and common-utils?

Reply:
Thanks a lot. I need to remove ml-client. It is a temporary hack for a bug with ml-client where they are exposing all the classes in common-utils with same package name. This is causing jar-hell issue. Now they have solved the issue, I will remove the dependency.